### DISCUSSION TITLE
Be able to have multiple status by project view

### DISCUSSION THREAD

**Discussion by xlr-8** [community/community#11302] [Projects and Issues]
Disclaimer: please let me know if a similar discussion already exist on it, I couldn't find it myself. Sorry

So currently we have some projects which are composed of 3 different teams: design, frontend & backend.

When working on a new feature, it doesn't always involve everyone but it would be nice to have a view centralizing all the team's status instead of grouping them individually. Here is how it is currently:

| TeamA | Status |
| --------- | --------- |
|  Issue 1 | TODO |
|  Issue 2 | TODO |
|  Issue 3 | DONE |
| **TeamB** | **Status** |
|  Issue 1| TODO|
| Issue 2 | DONE|
| Issue 3 | DONE|

Which makes it much harder to read than having:

| Issues| TeamA |  TeamB |
|----------|-----------|------------|
|Issue 1    | TODO |    TODO
|Issue 2   |  TODO  |   DONE
|Issue 3  |   DONE   |  DONE

By playing with the view we could then display only TeamA or TeamB.

**Comment by DanielOberlechner** [comment]
Hey I found your question via google and maybe a solution:
![Bildschirm­foto 2022-10-14 um 00 10 24](https://user-images.githubusercontent.com/10922763/195720389-7fdd9c39-6d50-4c91-8123-7b8c5b3ab225.png)

Here can't have 2 booleans in one sheet but maybe it helps you a bit further.

Kinds regards,
Daniel Oberlechner
